pkg/rpc/response/result/validator.go

```diff
@@ -8,7 +8,7 @@ import (
 // Validator used for the representation of
 // state.Validator on the RPC Server.
 type Validator struct {
-	PublicKey keys.PublicKey `json:"publickey"`
-	Votes     util.Fixed8    `json:"votes"`
-	Active    bool           `json:"active"`
+	PublicKey *keys.PublicKey `json:"publickey"`
```

Comment:
NACK, I don't think it's worth it. Add MarshalJSON for non-pointer structure and be done with it.